src/Core/DebugMessageCallback.cpp

```diff
@@ -0,0 +1,83 @@
+#include "Core/DebugMessageCallback.hpp"
+
+#include <string>
+
+#include "Util/Logger.hpp"
+
+namespace Core {
+void OpenGLDebugMessageCallback(GLenum source, GLenum type, GLuint id, // NOLINT
+                                GLenum severity, GLsizei length,
+                                const GLchar *message, const void *data) {
+    std::string sourceString;
+    std::string typeString;
+    std::string severityString;
+
+    // clang-format off
```

Comment:
Is disabling clang-format good?

Reply:
because I like everything to be aligned and it breaks the 80 line width limit
I don't think this is a big issue